### DISCUSSION TITLE
How can I set the emissive not overexposure

### DISCUSSION THREAD

**Discussion by fanbinnet** [google/filament#4511] [Q&A]
When I use emissive， the effect I can get like below, the color is overexposure become some white.
![overexposure1](https://user-images.githubusercontent.com/3060688/129686873-dcd492fd-f0f8-47cc-a506-d9341674d9e7.JPG)

The real effect I want is like below:

![rightexposure1](https://user-images.githubusercontent.com/3060688/129687152-b1add7e3-a88c-4d45-895b-0ff44a2425cd.JPG)

How can I get the effect which the color not overexposure ?

Thanks for any help!

**Comment by romainguy** [comment]
You can change the intensity of your emissive color to address this. See https://google.github.io/filament/Materials.html#materialmodels/litmodel/emissive for details. You can also use a different tone mapping operator like the one called `filmic` that won't roll-off to white on over-exposure (which I don't recommend as it's not correct, nor does it look good, and has side effects like hue skews at high intensities).

**Reply by fanbinnet** [reply]
I have set the bloom strength to max 1.0f, If I turn low the intensity of the emissive， the bloom effects will also became very low，
even can not be noticed.

Is there any more information about the filmic ? I will to study the filmic to if it can resolve my problem.

**Reply by romainguy** [reply]
If you want emissive to bloom you will by definition overexpose the camera sensor.

**Reply by fanbinnet** [reply]
Thanks, I think definition overexpose is the right method. But I don't know how to do this definition.
Can you give me some suggestion ?
